capabilities/code-maintainability.md

```diff
@@ -37,77 +56,91 @@ Don't worry if the description doesn't exactly match your situation. The descrip
 
 ### Previously Written Code
 
-1. **Brittle Codebase:** Changing any code is time-consuming, complex, and error prone.
-2. **Fairly Complex:** Most changes require significant refactoring, and it's difficult to predict the impact of changes on the overall system.
-3. **Partially Modular:** Most parts of the system are modular and easy to update, but some are complex and difficult to work with.
-4. **Well Organized:** When changes are made to the existing codebase, they don’t tend to require much rework.
+1. **Brittle Codebase:** Changing any code is time-consuming, complex, and prone to error.
+2. **Fairly Complex Codebase:** Most changes require significant refactoring, and it's difficult to predict the impact of changes on the overall system.
+3. **Partially Modular Codebase:** Most parts of the system are modular and easy to update, but some are complex and difficult to work with.
+4. **Well-organized Codebase:** When changes are made to the existing codebase, they don’t tend to require much rework.
+
+Now average your scores for the two responses and that will be your overall score for this capability.
+
+<ed> By "two responses" do we mean New Code and Previously Written Code? I think we could make this instruction clearer for the reader. My understanding is that the reader is to score themselves on each of the 4 areas in each of the two "subcategories" above (New Code and Previously Written Code). Do they then average the scores of each subcategory or do they average the scores of ALL areas (which would be 8 areas total)? </ed> 
 
-Now average the two responses and that will be your score for this capability.
+<ed> I suggest adding a line or two here that clarifies your vision for next steps for readers based on their score. For example, if their score was a 4, they're good to go here and can opt to move on to Related Capabilities? If their score was 3 or below, they should see Supporting Practices? </ed>
```

Comment:
Got it. Do you have any suggestions for the specific text? The basic idea here is a low score means there's probably a lot of value in experimenting with some of the supporting practices. If they scored highly, there's maybe not much of a reason to keep reading. They should probably look at other areas and try to raise those scores first.

Reply:
See edits in October. 30 commit. [PS. Is this the way we continue to have dialogue? On the original commit thread? Or shall I transfer open queries to the new commits?]